src/components/Images.vue

```diff
@@ -179,8 +248,9 @@ export default {
 			event.preventDefault()
 
 			// scrolling position relative to the image
-			const scrollX = event.clientX - this.$el.x - (this.width * this.zoomRatio / 2)
-			const scrollY = event.clientY - this.$el.y - (this.height * this.zoomRatio / 2)
+			const element = this.$refs.image ?? this.$refs.video
```

Comment:
can you zoom in the video?

Reply:
It's rather to allow to zoom into the poster of the video